src/Contrib/Jaeger/IdConverter.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Jaeger;
+
+use Brick\Math\BigInteger;
+
+/**
+ * An implementation of the section of the spec around converting OTEL's span/trace id representations to Thrift's format
+ * https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/jaeger.md#ids
+ */
+class IdConverter
+{
+    const HEX_STR_FOR_2_TO_THE_POWER_63 = '8000000000000000';
```

Comment:
How about the name `HEX_STR_FOR_2_POWER_63`?

Reply:
Yeah no preferences here. If anyone has a strong preference before this is merged I'll swap it in